docs/framework/winforms/creating-a-new-windows-form.md

```diff
@@ -19,7 +19,7 @@ ms.author: dotnetcontent
 manager: "wpickett"
 ---
 # Creating a New Windows Form
-This topic contains links to topics that describe how to create your first Windows Forms application. Also, the topics in this section introduce some of the basic vocabulary and guidelines that you should understand when you start to create a Windows Forms application. . To learn more about Windows Forms application; the controls you can use on them, events and handling events and how to handle input from the user see the related topic list.  
+This topic contains links to topics that describe how to create your first Windows Forms application. Also, the topics in this section introduce some of the basic vocabulary and guidelines that you should understand when you start to create a Windows Forms application. To learn more about Windows Forms application; the controls you can use on them, events and handling events and how to handle input from the user see the related topic list.  
   
```

Comment:
application --> applications,
events and handling events --> events and handling events,
from the user --> from the user,

Reply:
@rpetrusha Made those changes as suggested. :)